lib/scoreboard.js

```diff
@@ -4,38 +4,44 @@ const sortItems = (a, b) => {
   return 1
 }
 class ScoreBoard {
-  constructor(packet) {
+
+  constructor (packet) {
     this.name = packet.name
     this.title = packet.displayText
     this.itemsMap = {}
   }
-  add(name, value) {
-    this.itemsMap[name] = {
-      name,
-      value
-    }
+
+  add (name, value) {
+
+    this.itemsMap[name] = { name, value }
     return this.itemsMap[name]
   }
-  remove(name) {
+
+  remove (name) {
+
     const removed = this.itemsMap[name]
     delete this.itemsMap[name]
     return removed
   }
-  get() {
+
+  get () {
```

Comment:
This should be a getter of `items`, right now it's a `get` method

Reply:
right, fixed